test/built-ins/Array/prototype/sort/call-with-boolean.js

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2020 Rick Waldron.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.prototype.sort
+description: Array.prototype.sort applied to boolean primitive
+---*/
+
+assert.sameValue(Array.prototype.sort.call(true), true);
```

Comment:
Per https://github.com/tc39/ecma262/pull/1585, `? ToObject(this value)` is returned.

Reply:
I'll back this one out

Reply:
Filed https://bugs.chromium.org/p/v8/issues/detail?id=11362